Client/Frontend/Settings/Clearables.swift

```diff
@@ -84,26 +45,14 @@ class CookiesClearable: Clearable {
     
     func clear() -> Success {
         UserDefaults.standard.synchronize()
-        
         let result = Deferred<Maybe<()>>()
         // need event loop to run to autorelease UIWebViews fully
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
             // Now we wipe the system cookie store (for our app).
-            let storage = HTTPCookieStorage.shared
-            if let cookies = storage.cookies {
-                for cookie in cookies {
-                    storage.deleteCookie(cookie)
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: [WKWebsiteDataTypeCookies], modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
```

Comment:
i would expect local storage / websql / indexdb, etc., to be cleared when cookies are cleared (though i know this wasn't the case before). we have an open issue for this in brave-core. might as well do that here if it's easy.

Reply:
Added, thanks for the heads up

Reply:
thanks, here is the b-c issue for reference: https://github.com/brave/brave-browser/issues/1127